src/containers/RestaurantsList.js

```diff
@@ -0,0 +1,61 @@
+import React,{ Component } from 'react';
+import '../Styles.css'
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import {fetchRestaurants} from '../actions/Actions';
+import Button from '@material-ui/core/Button';
+import GridView from '../components/GridView.js';
+
+class RestaurantsList extends Component{
+
+    componentDidMount() {
+       this.props.getRestaurants();
+    }
+
+    render(){
+        const { restaurants, isFetching, lastUpdate, error } = this.props;
+        return(
+            <div className='Restaurants-list'>
+                <p>
```

Comment:
you can use [React.fragment](https://reactjs.org/docs/fragments.html) if you used the <p> just to bound  these components together.
```
<>
  isFetching
  error
  lastUpdate
</>
```

Reply:
if you meant the <p> so removed it